content/ko/docs/concepts/policy/pod-security-policy.md

```diff
@@ -0,0 +1,618 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name:      pause
+spec:
+  containers:
+    - name:  pause
+      image: k8s.gcr.io/pause
+EOF
+Error from server (Forbidden): error when creating "STDIN": pods "pause" is forbidden: unable to validate against any pod security policy: []
+```
+
+**무슨 일이 일어났나?** PodSecurityPolicy가 생성되었지만 파드의 서비스 어카운트나 `fake-user`는
+새 정책을 사용할 권한이 없다.
+
+```shell
+kubectl-user auth can-i use podsecuritypolicy/example
+no
+```
+
+예제 정책에서 `fake-user`에게 `use` 동사를 부여하는 rolebinding을 생성한다.
+
+{{< note >}}
+이 방법은 권장하지 않는다! 선호하는 방법은 [다음 절](#다른-파드를-실행)을
+참고하길 바란다.
+{{< /note >}}
+
+```shell
+kubectl-admin create role psp:unprivileged \
+    --verb=use \
+    --resource=podsecuritypolicy \
+    --resource-name=example
+role "psp:unprivileged" created
+
+kubectl-admin create rolebinding fake-user:psp:unprivileged \
+    --role=psp:unprivileged \
+    --serviceaccount=psp-example:fake-user
+rolebinding "fake-user:psp:unprivileged" created
+
+kubectl-user auth can-i use podsecuritypolicy/example
+yes
+```
+
+이제 파드 생성을 다시 시도하자.
+
+```shell
+kubectl-user create -f- <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name:      pause
+spec:
+  containers:
+    - name:  pause
+      image: k8s.gcr.io/pause
+EOF
+pod "pause" created
+```
+
+예상대로 작동한다! 그러나 특권있는 파드를 만들려는 시도는 여전히 거부되어야 한다.
+
+```shell
+kubectl-user create -f- <<EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name:      privileged
+spec:
+  containers:
+    - name:  pause
+      image: k8s.gcr.io/pause
+      securityContext:
+        privileged: true
+EOF
+Error from server (Forbidden): error when creating "STDIN": pods "privileged" is forbidden: unable to validate against any pod security policy: [spec.containers[0].securityContext.privileged: Invalid value: true: Privileged containers are not allowed]
+```
+
+계속 진행하기 전에 파드를 삭제하자.
+
+```shell
+kubectl-user delete pod pause
+```
+
+### 다른 파드를 실행
+
+약간 다르게 다르게 다시 시도해보자.
+
+```shell
+kubectl-user run pause --image=k8s.gcr.io/pause
+deployment "pause" created
+
+kubectl-user get pods
+No resources found.
+
+kubectl-user get events | head -n 2
+LASTSEEN   FIRSTSEEN   COUNT     NAME              KIND         SUBOBJECT                TYPE      REASON                  SOURCE                                  MESSAGE
+1m         2m          15        pause-7774d79b5   ReplicaSet                            Warning   FailedCreate            replicaset-controller                   Error creating: pods "pause-7774d79b5-" is forbidden: no providers available to validate pod request
+```
+
+**무슨 일이 일어났나?** 우리는 이미 `fake-user`에 대해 `psp:unprivileged` 역할을 바인딩했는데
+`Error creating: pods "pause-7774d79b5-" is
+forbidden: no providers available to validate pod request` 오류가
+발생하는 이유는 무엇인가? 그 답은 소스인 `replicaset-controller`에 있다. `fake-user`가
+디플로이먼트를 성공적으로 생성했지만(레플리카셋을 성공적으로 생성했음) 레플리카셋이
+파드를 생성했을 때 podsecuritypolicy 예제를
+사용할 권한이 없었다.
+
+이 문제를 해결하려면 `psp:unprivileged` 역할을 파드의 서비스 어카운트에 대신
+바인딩한다. 이 경우(지정하지 않았으므로) 서비스 어카운트는 `default`이다.
```

Comment:
> origin
> account instead. In this case (since we didn't specify it) the service account	
> is `default`:	

```suggestion
바인딩한다. 이 경우(지정하지 않았으므로) 서비스 어카운트는 
`default`이다.
```

- [ ] 원문에 따라 개행이 필요합니다.